knative-operator/deploy/resources/knativekafka/source/eventing-kafka-post-install.yaml

```diff
@@ -198,3 +198,137 @@ spec:
+  - apiGroups:
+      - "sources.knative.dev"
+    resources:
+      - "kafkasources"
+      - "kafkasources/finalizers"
+      - "kafkasources/status"
+    verbs:
+      - "get"
+      - "list"
+      - "create"
+      - "update"
+      - "patch"
+      - "watch"
+  - apiGroups:
+      - ""
+    resources:
+      - "namespaces"
+    verbs:
+      - "get"
+      - "list"
+---
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: knative-kafka-storage-version-migrator
+  namespace: knative-eventing
+  labels:
+    kafka.eventing.knative.dev/release: devel
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: knative-kafka-storage-version-migrator
+  labels:
+    kafka.eventing.knative.dev/release: devel
+subjects:
+  - kind: ServiceAccount
+    name: knative-kafka-storage-version-migrator
+    namespace: knative-eventing
+roleRef:
+  kind: ClusterRole
+  name: knative-kafka-storage-version-migrator
+  apiGroup: rbac.authorization.k8s.io
+---
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: knative-kafka-storage-version-migrator
+  namespace: knative-eventing
+  labels:
+    app: "knative-kafka-storage-version-migrator"
+    kafka.eventing.knative.dev/release: devel
+spec:
+  ttlSecondsAfterFinished: 600
+  backoffLimit: 10
+  template:
+    metadata:
+      labels:
+        app: "knative-kafka-storage-version-migrator"
+        kafka.eventing.knative.dev/release: devel
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: knative-kafka-storage-version-migrator
+      restartPolicy: OnFailure
+      containers:
+        - name: migrate
+          image: registry.ci.openshift.org/openshift/knative-v1.1.0:knative-eventing-kafka-storage-version-migration
```

Comment:
Can we add a TODO here or capture somewhere else this, we will need the `eventing-kafka-broker` build eventually once we no longer build `eventing-kafka`, right?

Reply:
yeah, or the one from eventing - will capture something